packages/amazonq/src/lsp/activation.ts

```diff
@@ -5,13 +5,13 @@
 
 import vscode from 'vscode'
 import { startLanguageServer } from './client'
-import { AmazonQLSPResolver } from './lspInstaller'
+import { AmazonQLSPInstaller } from './lspInstaller'
```

Comment:
When an acroynym has 3 or more chars, its camelcase form is `Xxx`, not `XXX`. Since we use camelcase, it's useful to be consistent. E.g. HTML is Html in camelcase.